setup.py

```diff
@@ -319,6 +319,15 @@ def run(self):
 # - `ops`: Path to the compiler operations module.
 # - `qjit`: Path to the JIT compiler decorator provided by the compiler.
 
+# Install the `catalyst` binary into the user's Python environment so it is accessible on the PATH.
+# Does not work with editable installs. Requires the Catalyst mlir module to be built.
+if os.path.exists("frontend/catalyst/bin/catalyst"):
```

Comment:
```suggestion
if pathlib.Path("frontend", "catalyst", "bin", "catalyst").exists()
```
I'd recommend using the system agnostic pathlib with this and below.